addons/account_edi/models/account_move.py

```diff
@@ -144,59 +360,18 @@ def _prepare_edi_vals_to_export(self):
         '''
         self.ensure_one()
 
-        def convert(amount):
-            return self.currency_id._convert(amount, self.company_currency_id, self.company_id, self.date)
-
         res = {
             'record': self,
+            'sign': -1 if self.is_inbound() else 1,
```

Comment:
Maybe a clearer name can be found ? Without context, and in the middle of a template, knowing what 'sign' means might be a bit hard.